### ISSUE TITLE
Combine the "core owncloud calendar app" with @libasys app

### ISSUE THREAD

**Issue by farson2003** [owncloud/calendar#805]
Would it not be good to combine the "core owncloud calendar app" with @libasys app here, which provides public link sharing, instead of having multiple different calendar apps? https://github.com/libasys/ownCloud-8.1-Calendar-App
@georgehrke @jancborchardt 

Not sure what the underlying policies are...

**Comment by jancborchardt** [comment]
Yes, I would love to have us work on one Calendar app. @georgehrke will be able to devote more time to the Calendar so we should see how to handle that. We need to see what to do with the rework branch, with the outstanding pull requests, and with @libasys's fork.

I invite you all to join our IRC channel at #owncloud-calendar so we can properly coordinate work and talk about it. :)

**Comment by jancborchardt** [comment]
Yes, I would love to have us work on one Calendar app. @georgehrke will be able to devote more time to the Calendar so we should see how to handle that. We need to see what to do with the rework branch, with the outstanding pull requests, and with @libasys's fork.

I invite you all to join our IRC channel at #owncloud-calendar so we can properly coordinate work and talk about it. :) cc @DeepDiver1975 @tomneedham as well.

**Comment by jancborchardt** [comment]
Yes, I would love to have us work on one Calendar app. @georgehrke will be able to devote more time to the Calendar so we should see how to handle that. We need to see what to do with the rework branch, with the outstanding pull requests, and with @libasys's fork.

I invite you all to join our IRC channel at #owncloud-calendar so we can properly coordinate work and talk about it. :) cc @DeepDiver1975 @raghunayyar @tomneedham as well.

**Comment by Tramour** [comment]
Libasys' calendar app offers a lot of features I was looking for. Shared calendars can be turned off, integration of tasks, nicer GUI, pop-over-information, adjustible colors for calendars...
Any news from here what the new calendar app for 8.1 will be?
